packages/loopring_v3/contracts/impl/libexchange/ExchangeBlocks.sol

```diff
@@ -122,8 +122,8 @@ library ExchangeBlocks
                 "PROOF_TOO_LATE"
             );
 
-            // Maybe we should strip the highest bits of the public input so we don't have any overflow (uint/prime field)
-            publicInputs[i] = uint(specifiedBlock.publicDataHash);
+            // Strip the highest bits of the public input so we don't have any overflow in the snark field
+            publicInputs[i] = uint(specifiedBlock.publicDataHash) >> 3;
```

Comment:
The team "highest bits" is a bit confusing. If you meant the most significant bits, you should use:
`(uint(publicDatahash) << 3) >>3`; if you meant the least significant bits, then your implementation is correct.

Reply:
Ah yes, the comment here is wrong, will fix.